### PR TITLE
Add new greeting.

### PR DESCRIPTION
**Pull request by Ant-C-tech**

### CODE REVIEW

**Review by lenadgit — CHANGES_REQUESTED** [review]
Good job!  But if you look at your file  https://github.com/kottans/mock-repo/pull/548/files  you will see the highlighted lines in red. It happened because your code-editor autoformatted lines (indentations, etc), and we can not merge it. Could you please fix it? How to resolve this issue - was mentioned here 
https://github.com/kottans/mock-repo/pull/460#pullrequestreview-518209900

**Comment by Ant-C-tech** [comment]
Please, check my greeting)

File: README.md [hunk 1]
```diff
@@ -1832,3 +1832,7 @@ My name is [Igor](https://github.com/ik-web) and I'm very happy to join the Kott
 
 ![Ninja](assets/images/ninja.jpg)
 
+
+My name is [Anton] (https://github.com/Ant-C-tech). First part of education course "Version Control with Git" and especially "learngitbranching.js.org" was very useful for me and now I'm excited to continue adventure with your wonderful interesting community.
```
Comment (miroslav-kolomiets) [inline README.md:1836]:
```suggestion
My name is [Anton](https://github.com/Ant-C-tech). First part of education course "Version Control with Git" and especially "learngitbranching.js.org" was very useful for me and now I'm excited to continue adventure with your wonderful interesting community.
```

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Hi! Good job! But you have a typo, that's why your link looks like
Снимок экрана 2020-11-12 в 9 43 51 PM

**Comment by lenadgit** [comment]
Good. But as you see - your file now has a conflict. If you  look at the file, you will see that other participant added changes after your greeting 
https://github.com/kottans/mock-repo/pull/548/conflicts
How to resolve it:
Type at command line:
- git pull origin main
- git push
You can get new changes (Anastasia's greeting) and push your changes again

**Review by miroslav-kolomiets — APPROVED** [review]
Nice job!

**Comment by Ant-C-tech** [comment]
Can you help me to understand this moment? In order if there are many pull requests a day you can merge only one of them, because others would be need some revision? When I made my last  pull request there was no commit #550. If I understand it right, commit #550 was in queue for merge. I would like to understand how can one do merging without revision all the queue of pull requests? It would help me to understand how can I do such things properly. Thank you.

**Comment by miroslav-kolomiets** [comment]
> Can you help me to understand this moment? In order if there are many pull requests a day you can merge only one of them, because others would be need some revision? When I made my last pull request there was no commit #550. If I understand it right, commit #550 was in queue for merge. I would like to understand how can one do merging without revision all the queue of pull requests? It would help me to understand how can I do such things properly. Thank you.

If I understand your question correctly:

These things do not depend on each other, only if the changes concerned the same places in the code. In this case, you need to resolve the conflict. In other cases, commits, with merge, do not affect each other and doesn't need some special revision.

**Comment by Ant-C-tech** [comment]
Thank you for the answer. I mean, for example, today you have 20 commits and each wants to start on line 1825 (last line in Readme. md). If you merge one of them other commits need revision because line 1825  is now occupied. This case for me is interesting.

**Pull request merged**